grails-app/services/org/transmartproject/db/querytool/QueriesResourceService.groovy

```diff
@@ -169,12 +171,12 @@ class QueriesResourceService implements QueriesResource {
     }
 
     @Override
-    List<QueryResult> getQueryResultsByUsername(String username) {
+    List<QueryResultSummary> getQueryResultsByUsername(String username) {
         def query = QtQueryResultInstance.where {
             queryInstance.userId == username
         }
-        query.findAll()
-    }
+        query.findAll().collect { new QueryResultSummaryImplementation(it)}
```

Comment:
Any idea why `findAll()` is used here. Could it be simply `query.collect { new QueryResultSummaryImplementation(it)}` ?